### ISSUE TITLE
Fill and Dig options

### ISSUE THREAD

**Issue by kweeniemike** [bw2012/UnrealSandboxTerrain#3]
Was trying to add the option to use X, Y, Z sizes instead of the radius so you could use a beam of 1x2x4 for instance for digging, but found out that radius is tied to a lot of stuff. So I am not sure if this should be in Issues or somewhere else, but the possibility to use a beam of size X by Y by Z would be a nice option to have. 

Was also trying to see how to do this but with a direction Vector to it (So a beam facing a certain direction instead of just straight up or down) but that is probably a whole new realm of problems.

**Comment by bw2012** [comment]
Yep, unfortunately design of this part is not so good. I'll refactor it soon and add feature to edit terrain with 3 dimensions box

**Comment by mhoangvslev** [comment]
I would like to suggest several point:
- Is it possible to build the terrain in the editor? 
- If not can you make the default terrain destructible? I am only looking for partial destruction on terrain created by the editor.
